### PR TITLE
feat: Support deletion of Billing Account Log Sinks

### PR DESCRIPTION
**Pull request by daniel-cit**
This PR supports the deletion of Billing Account Log Sinks older then a given age and that match a given pattern.

### CODE REVIEW

**Comment by daniel-cit** [comment]
@apeabody Could you PTAL?

File: modules/project_cleanup/README.md [hunk 1]
```diff
@@ -12,34 +12,41 @@ Running this module requires an App Engine app in the specified project/region.
...
 
+- Artifact Registry API (`artifactregistry.googleapis.com`)
 - Cloud Functions (`cloudfunctions.googleapis.com`)
 - Cloud Scheduler (`cloudscheduler.googleapis.com`)
 - Cloud Resource Manager (`cloudresourcemanager.googleapis.com`)
 - Compute Engine API (`compute.googleapis.com`)
 - Cloud Asset API (`cloudasset.googleapis.com`)
 - Security Command Center API (`securitycenter.googleapis.com`)
+- Cloud Logging API (`logging.googleapis.com`)
+
```
Comment (apeabody) [inline modules/project_cleanup/README.md:23]: nit: extra line?
```suggestion
```

File: modules/project_cleanup/function_source/main.go [hunk 2]
```diff
@@ -302,34 +300,13 @@ func getSCCNotificationPageSizeOrTerminateExecution() int32 {
...
-	targetIncludedFeeds := os.Getenv(envVariableName)
-	logger.Println("Try to get CAI Feeds list")
-	if targetIncludedFeeds == "" {
-		logger.Printf("No CAI Feeds provided.")
-		return compiledRegEx
-	}
-
-	var caiFeeds []string
-	err := json.Unmarshal([]byte(targetIncludedFeeds), &caiFeeds)
+func getBillingSinksPageSizeOrTerminateExecution() int64 {
```
Comment (apeabody) [inline modules/project_cleanup/function_source/main.go:303]: Would it make sense to convert to a generic `getIntFromEnv()`?

**Review by apeabody — APPROVED** [review]
Thanks @daniel-cit!

**Pull request merged**